packages/salesforcedx-vscode-soql/package.json

```diff
@@ -83,7 +83,10 @@
     "watch": "tsc -watch -p .",
     "clean": "shx rm -rf node_modules && shx rm -rf out && shx rm -rf coverage && shx rm -rf .nyc_output",
     "test": "npm run test:vscode-integration",
-    "test:ui": "extest setup-and-run -s ~/.vscode-test-resources -o test/ui-test/resources/settings.json -e test/ui-test/resources/extensions out/test/ui-test/*.test.js",
+    "test:ui:prepare": "extest get-vscode -s ~/.vscode-test-resources && extest get-chromedriver -s ~/.vscode-test-resources ",
+    "test:ui:install": "extest install-vsix -f ./salesforcedx-vscode-soql-56.2.0.vsix -s ~/.vscode-test-resources -e test/ui-test/resources/extensions ",
+    "test:ui:run": "extest run-tests -s ~/.vscode-test-resources -o test/ui-test/resources/settings.json -e test/ui-test/resources/extensions out/test/ui-test/*.test.js",
+    "test:ui": "npm run test:ui:prepare && npm run test:ui:install && npm run test:ui:run",
```

Comment:
SO previously the extest command would call vsce and create a vsix file.  This is no longer possible without our bundle/package wrapper so I just updated it here to do the different pieces individually instead of using the setup-and-run task.  This works locally, but will need to verify in Circle after merge as the test:ui job only runs as part of the scheduled build.